.gitignore

```diff
@@ -20,4 +20,4 @@ yarn-error.log*
 
 
 #TODO add .env to your gitignore
-
+.env
```

Comment:
Let's add this after @tsyaeger also adds their name

Reply:
Okay, sorry about that!